src/components/content/heroes/hero-text.tsx

```diff
@@ -28,6 +28,7 @@ const HeroTextStyled = styled.div`
 `;
 
 const Headline = styled.h1`
+  white-space: pre-line;
```

Comment:
Nice that you leverage this css property. When this feature was discussed I already had br tags in the templates in mind and inject potentially unsafe html tags. I like the solution but I have some respect about the consequences. Do we have some disadvantages by swapping br with newlines? Or is it just equal and I learned some technique from you to use in the future 😄🙌